tests/fsharp/Compiler/ErrorMessages/SuggestionsTests.fs

```diff
@@ -0,0 +1,410 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace FSharp.Compiler.UnitTests
+
+open NUnit.Framework
+open FSharp.Compiler.SourceCodeServices
+
+[<TestFixture>]
+module Suggestions =
+
+    [<Test>]
+    let ``Field Suggestion`` () =
+        CompilerAssert.TypeCheckSingleError
+            """
+type Person = { Name : string; }
+
+let x = { Person.Names = "Isaac" }
+    
+exit 0
+            """
+            FSharpErrorSeverity.Error
+            39
+            (4, 18, 4, 23)
+            "Maybe you want one of the following:\nName"
+
+
+    //TODO This was marked with SCFLAGS="--vserrors" in tests/fsharpqa/Source/Warnings/env.lst file
```

Comment:
Please take note on this kind of comments. There is anything special we need to do in this case?

Reply:
I don't think so, no.

Reply:
Then I should remove all those comments? I put them there to not lose info on the transition (since I deleted the specific rows from env.lst)

Reply:
I think it's fine to delete, especially since they're specific to that testing environment. And since the tests pass in CI, it seems they're fine!